server/src/api/community.rs

```diff
@@ -417,42 +420,89 @@ impl Perform for Oper<EditCommunity> {
       Err(_e) => return Err(APIError::err("couldnt_update_community").into()),
     };
 
-    // Mod tables
-    if let Some(removed) = data.removed.to_owned() {
-      let expires = match data.expires {
-        Some(time) => Some(naive_from_unix(time)),
-        None => None,
-      };
-      let form = ModRemoveCommunityForm {
-        mod_user_id: user_id,
+    // TODO there needs to be some kind of an apub update
+    // process for communities and users
```

Comment:
Well its automatically refetched after 24 hours. Sending out an update activity isnt exactly required, but might be a nice bonus. Should probably have an issue for that (unless we do and I cant find it).